components/aiprompt/toolbar.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Toolbar
+page_title: AIPrompt - Toolbar
+description: Configure and use the Blazor AIPrompt toolbar and its buttons. How to define custom toolbar buttons.
+slug: aiprompt-toolbar
+tags: telerik,blazor,aiprompt,ai,prompt,toolbar
+published: True
+position: 20
+---
+
+# AIPrompt Toolbar
+
+By default, the AIPrompt toolbar will always render the buttons corresponding to the currently visible predefined views (Prompt, Output, and optionally, Command). When clicked, these built-in buttons will activate the appropriate view they represent. This article shows how to add custom tools to the AIPrompt's toolbar, which may invoke arbitrary commands and/or handlers.
+
+
+## Custom Tools
```

Comment:
This section should clarify where custom buttons appear - always after the built-in buttons? Does the developer have control over this?

Reply:
Indeed, they will appear right after all the views. I'll add a note.